eng/scripts/Create-ApiReview.ps1

```diff
@@ -99,16 +99,18 @@ function ProcessPackage($PackageName)
                 return 1
             }
 
+            $revisionLabel = "$($pkgInfo.Version) - Source Branch:$($SourceBranch)"
```

Comment:
Does this require changes on the APIView side? Do we every make an assumption that this is only a version anywhere?

Reply:
```suggestion
            $revisionLabel = "$($pkgInfo.Version) - Source Branch:${SourceBranch}"
```

Reply:
Package version is part of token Code file itself. Label param in REST API to create a review revision is only used to create the label and it's not parsed or treated as package version. This param can have any text as value.